spec/lib/api/v3/utilities/path_helper_spec.rb

```diff
@@ -97,6 +97,14 @@
     it { is_expected.to match(/^\/api\/v3\/categories\/42/) }
   end
 
+  describe '#create_work_package_form' do
+    subject { helper.create_work_package_form 42 }
+
+    it_behaves_like 'api v3 path'
+
+    it { is_expected.to match(/^\/api\/v3\/projects\/42\/work_packages\/form/) }
```

Comment:
didn't we agree that strings are easier to read than regexes?

Especially since I believe the following string would pass this test:

````
/api/v3/projects/42/work_packages/form/dinosaurs/have/taken/control, I can write whatever I want!!!!

Probably also line breaks!
````